### PR TITLE
chore: migrate nuxt module to HMR version

### PR DESCRIPTION
**Pull request by Szymon-dziewonski**
# Related issue

Closes #

# Scope of work

Newest version `@nuxtjs/tailwindicss:6.12.1` that supports HMR our module has to migrated 

Connected GH issue https://github.com/nuxt-modules/tailwindcss/issues/898#event-14528876142

# Screenshots of visual changes

# Checklist

- [ ] Self code-reviewed
- [ ] Changes documented
- [ ] Semantic HTML
- [ ] SSR-friendly
- [ ] Caching friendly
- [ ] a11y for WCAG 2.0 AA
- [ ] examples created
- [ ] blocks created
- [ ] cypress tests created

### CODE REVIEW

File: apps/docs/components/package.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     "@microsoft/api-documenter": "^7.25.9",
     "@microsoft/api-extractor": "^7.47.4",
     "@types/node": "^20.12.7",
-    "nuxt": "^3.12.4",
-    "nuxt-gtag": "^2.1.0"
+    "nuxt": "^3.13.2",
+    "nuxt-gtag": "^2.0.5"
```
Comment (dawid-ziobro) [inline apps/docs/components/package.json]: Is the downgrade on purpose?

Comment (Szymon-dziewonski) [reply apps/docs/components/package.json]: I havent changed this version of nuxt-gtag, weird, yarn had to change it, good catch will check it

Comment (Szymon-dziewonski) [reply apps/docs/components/package.json]: https://github.com/vuestorefront/storefront-ui/pull/3232/commits/79406be304830a435cdf725fd63edc7e29ab6290

Comment (Szymon-dziewonski) [reply apps/docs/components/package.json]: https://github.com/vuestorefront/storefront-ui/pull/3232/commits/79406be304830a435cdf725fd63edc7e29ab6290

File: packages/sfui/frameworks/nuxt/playground/tailwind.config.ts [hunk 2]
```diff
@@ -1,9 +1,16 @@
+// How to test? check if variable is overwritten with current priority table
+// 1. `tailwind.config.ts` file
+// 2. `nuxt.config.ts` file with `tailwindcss` property
+// 3. default configuration inside module
```
Comment (FRSgit) [inline packages/sfui/frameworks/nuxt/playground/tailwind.config.ts:4]: :+1:

**Review by FRSgit — APPROVED** [review]

**Pull request merged**